.github/workflows/test_and_deploy.yml

```diff
@@ -0,0 +1,30 @@
+name: Build, Test, and Deploy
+
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Use Node.js 10.x
+      uses: actions/checkout@v1
+      with:
+        node-version: 10.x
```

Comment:
Should we bump it to 12, since that's the latest LTS?

Reply:
Op... saw this comment after merging.  Will do.